src/simplify_algebra.cpp

```diff
@@ -439,6 +444,39 @@ struct find_splits
         return groups;
     }
 
+    std::vector<int64_t> get_reduce_axes(instruction_ref ins) const
+    {
+        assert(contains(ins->name(), "reduce"));
+        std::vector<int64_t> axes;
```

Comment:
This should just read the axes with `ins->get_operator().to_value()["axes"].to_vector<int64_t>()`.